src/workflows-app/RunDetails.test.js

```diff
@@ -732,4 +733,19 @@ describe('BaseRunDetails - render smoke test', () => {
     // Ensure the diff is rendered
     screen.getByText('Result: View cache diff');
   });
+
+  it('should create the correct link when the execution directory button is clicked', async () => {
+    const user = userEvent.setup();
```

Comment:
Awesome test! Is there a sister test that uses a link like this to open some nested subdirectory?

Reply:
Is the test on line 1494 of SubmissionDetails.test.js not sufficient for this case? I think that putting a test here for that would be a copy paste of the one used for Submission Details